changelog.d/cache-remap-compilation.enhancement.md

```diff
@@ -0,0 +1,3 @@
+Cache result of remap compilation which is called multiple times during vector initialization
```

Comment:
Thanks for this. Ideally this should explain why it's an enhancement. i.e. Why should we care that this changed? In this case it is to improve startup/reload performance with large remap transforms.